### PR TITLE
Show amount due if available

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Show amount due if available

Amount due will show for unpaid invoices
Otherwise show total for paid

[stage-19]

## Motivation and Context
Fix for #2321

## How Has This Been Tested?
Tested changes locally with various invoices.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez please review. Thanks.

File: web/partials/billing/app-billing.html [hunk 1]
```diff
@@ -124,7 +124,7 @@ <h1 class="app-header-title">Account & Billing</h1>
               Invoice #{{item.invoice.id}}
             </td>
             <td class="table-body__cell">
-              ${{item.invoice.total / 100 | number:2}}
+              ${{(item.invoice.total || item.invoice.amount_due) / 100 | number:2}}
```
Comment (santiagonoguez) [inline web/partials/billing/app-billing.html:127]: @alex-deaconu is the order important ? in the card you mentioned ammount_due || total, but here it's the other way around, thanks !

Comment (alex-deaconu) [reply web/partials/billing/app-billing.html:127]: @santiagonoguez good catch, it should be reversed as per Trello. Will fix. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez reversed the logic. I tested by adding a credit note to this invoice `6459` in the Rise Vision test company, you'll need to scroll once through the list to see it. Original amount was `$247.50` but with the credit note it should only be `$200.00`. Thanks.

**Review by santiagonoguez — APPROVED** [review]

**Review by ezequielc — APPROVED** [review]

**Pull request merged**